src/util/SpawnObject.cpp

```diff
@@ -3,7 +3,36 @@
 
 namespace YimMenu
 {
-	void SpawnObject(uint32_t hash, Vector3 coords)
+	void PreviewObject(uint32_t hash, Vector3 coords, float pitch, float yaw, float roll, float alpha, bool isEnabled) 
+	{
+		FiberPool::Push([=] {
+			if (STREAMING::IS_MODEL_IN_CDIMAGE(hash) && STREAMING::IS_MODEL_VALID(hash))
+			{
+				if (!STREAMING::HAS_MODEL_LOADED(hash))
```

Comment:
Use a for loop here to add a timeout if the model doesn't load on the first try.